tsconfig.test.json

```diff
@@ -5,7 +5,7 @@
   "files": [],
   "include": [],
   "references": [
-    { "path": "./packages/integration-testsuite/" },
+    { "path": "./packages/integration-testsuite/tsconfig.cjs.json" },
```

Comment:
I wonder if this line is even necessary — I'd expect you to end up building integration-testsuite because it's referenced by tests later? Though it probably doesn't hurt.

Reply:
[991e32c](https://github.com/apollographql/apollo-server/pull/7055/commits/991e32c74909dee48bc6e2c7337f12a1df2568bd)